src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsParameters.java

```diff
@@ -32,23 +32,20 @@
 //END_TIGER_EXCLUDE
 
```

Comment:
These TIGER EXCLUDE comments could be removed at this stage.

Reply:
TIGER comments removed in commit https://github.com/openjdk/jdk/pull/610/commits/45f451829c30339a10f03377a78b17b7291be82f